src/ContainerRenameModal.jsx

```diff
@@ -58,7 +58,7 @@ const ContainerRenameModal = ({ container, version, updateContainerAfterEvent })
     const handleKeyPress = (event) => {
         if (event.key === "Enter") {
             event.preventDefault();
-            handleRename(name);
+            handleRename();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1207-20230216-111025-7f74f495-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)